tools/metro-extra-config/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@rnx-kit/metro-config": "^1.3.2",
     "@rnx-kit/metro-resolver-symlinks": "0.1.32",
+    "json5": "^2.2.3",
```

Comment:
Needed to read the `tsconfig.json` file with comments.

Reply:
do we really want comments on the json file ? it's not standard

Reply:
removed